src/SagaTester.js

```diff
@@ -13,7 +13,7 @@ const makeResettable = (reducer, initialStateSlice) => (state, action) => {
 export const resetAction = { type : RESET_TESTER_ACTION_TYPE };
 
 export default class SagaIntegrationTester {
-    constructor({initialState = {}, reducers, middlewares = []}) {
+    constructor({initialState = {}, reducers, middlewares = [], combineReducer = reduxCombineReducers}) {
```

Comment:
Hi,

Note you had a typo here - combineReducer -> combineReducers. Fixed in master.